ImageKitSample/Program.cs

```diff
@@ -0,0 +1,385 @@
+﻿using Imagekit.Models;
+using Imagekit.Sdk;
+using System;
+using System.Collections.Generic;
+using Imagekit;
+using static Imagekit.Models.CustomMetaDataFieldSchemaObject;
+using Newtonsoft.Json.Linq;
+
+namespace ImagekitSample
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            //Create Instance of ImageKit
+            ImagekitClient imagekit = new ImagekitClient("TestPublicKey", "TestPrivateKey", "https://api.imagekit.io/");
+
+
+            #region URL Generation
```

Comment:
What is region and how does it look to the user?

Reply:
Region to separate the code in c# and we can collapse and expand the section.